services/app-api/src/handlers/migrate_rate_documents.ts

```diff
@@ -0,0 +1,152 @@
+            'Configuration Error: REACT_APP_OTEL_COLLECTOR_URL must be set'
+        )
+    }
+    initMeter(serviceName)
+    for (const revision of revisions) {
+        const pkgID = revision.pkgID
+        const decodedFormDataProto = toDomain(revision.formDataProto)
+        if (!(decodedFormDataProto instanceof Error)) {
+            const formData = decodedFormDataProto as HealthPlanFormDataType
+            // skip the submission with two rates
+            if (formData.id !== 'ddd5dde1-0082-4398-90fe-89fc1bc148df') {
+                // we know the other submissions have only a single rate document
+                const ratesRelatedDocument = formData.documents.filter((doc) =>
+                    doc.documentCategories.includes('RATES_RELATED')
+                )
+                formData.rateInfos[0].supportingDocuments = ratesRelatedDocument
+                formData.documents = formData.documents.filter(
+                    (doc) => !ratesRelatedDocument.includes(doc)
+                )
+            } else {
+                // now handle the submisson with two rates
+                const firstRateRelatedDocument = formData.documents.filter(
+                    (doc) =>
+                        doc.name ===
+                        'Report12 - SFY 2022 Preliminary MississippiCAN Capitation Rates - Exhibits.xlsx'
+                )
+                const secondRateRelatedDocument = formData.documents.filter(
+                    (doc) =>
+                        doc.name ===
+                        'Report13 - SFY 2023 Preliminary MississippiCAN Capitation Rates - Exhibits.xlsx'
+                )
+                formData.rateInfos[0].supportingDocuments =
+                    firstRateRelatedDocument
+                formData.rateInfos[1].supportingDocuments =
+                    secondRateRelatedDocument
+                formData.documents = formData.documents.filter(
+                    (doc) =>
+                        !firstRateRelatedDocument.includes(doc) &&
+                        !secondRateRelatedDocument.includes(doc)
+                )
+            }
+            try {
+                const update = await store.updateHealthPlanRevision(
+                    pkgID,
+                    revision.id,
+                    formData
+                )
+                if (isStoreError(update)) {
+                    console.error(
+                        `StoreError updating revision ${
+                            revision.id
+                        }: ${JSON.stringify(update)}`
+                    )
+                    throw new Error('Error updating revision')
+                }
+            } catch (err) {
+                console.error(`Error updating revision ${revision.id}: ${err}`)
+                throw err
+            }
+        } else {
+            const error = `Error decoding formDataProto for revision ${revision.id} in rate migration: ${decodedFormDataProto}`
+            console.error(error)
```

Comment:
Could we include the `pkgID` in the error message too? If there was an error, it would be difficult to find which package this revision was from using the revision ID.

Reply:
good call.  done